src/rgw/rgw_http_client.h

```diff
@@ -148,6 +148,83 @@ class RGWHTTPHeadersCollector : public RGWHTTPClient {
 };
 
 
+#if 1
```

Comment:
I think this was because `#define dout_subsys ceph_subsys_rgw` appeared in the header due to the code movement. It was correct later, in a commit without *ONLY move (...)*.